main.go

```diff
@@ -114,16 +123,10 @@ func main() {
 		// basePath. After that, logs will go only to stderr.
 		fileOnlyLogger.Printf("Dumped system information to: %s", basePath)
 
-		var stdout, stderr bytes.Buffer
-
-		cmd := exec.Command("tar", "-czf", basePath+".tar.gz", basePath)
-		cmd.Stdout = &stdout
-		cmd.Stderr = &stderr
-		if err := cmd.Run(); err != nil {
-			log.Printf("There was an error creating archiving the dumped data, so the dumped system information is stored unarchived in: %s", basePath)
+		if err := archive(basePath); err != nil {
+			log.Printf("Could not archive dump data, unarchived data in: %s", basePath)
```

Comment:
This looks like better output to me, I wonder would there be value in outputting the error that was returned from the archive function?